python/ql/src/experimental/semmle/python/frameworks/StdLib.qll

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
If you want to take a file to the status of other branch, you can do `git checkout {branch-name} -- filename`. Then git won't show it in `Files changed` since there won't be any changes.

Reply:
And I think it is `Stdlib.qll` instead of `Std(L)ib.qll`. Fixing that detail now ;)

Reply:
Okay I get the weird issue now. There's no change in `Stdlib.qll` but there is in `StdLib.qll` although that file doesn't exist... 👽

Reply:
I think there is a solution to this. If we delete the file, make the git commit, then reintroduce the file with the correct name, it will work.

Reply:
Does `StdLib.qll` show up in your filesystem? If so, that makes sense 👍

Reply:
Taking this off GH since it's irrelevant to this PR.